.github/workflows/request-cli-integ-test.yml

```diff
@@ -27,13 +27,6 @@ jobs:
               - packages/aws-cdk/bin/**
               - packages/aws-cdk/lib/**
               - packages/aws-cdk/test/**
-              - packages/cdk-assets/bin/**
-              - packages/cdk-assets/lib/**
-              - packages/cdk-assets/test/**
-              - packages/aws-cdk-lib/cloud-assembly-schema/lib/**
-              - packages/aws-cdk-lib/cloud-assembly-schema/schema/**
-              - packages/aws-cdk-lib/cloud-assembly-schema/scripts/**
-              - packages/aws-cdk-lib/cloud-assembly-schema/test/**
```

Comment:
Cloud assembly schema is removed from here (and a few other places), but this PR doesn't delete that package and the title doesn't suggest that we're using the externally published version of cloud-assembly-schema as well. Can we update the PR title to reflect that?

Reply:
Cloud assembly schema was removed in a previous PR.